TA_dataset/bin/dataset_api.py

```diff
@@ -102,17 +106,25 @@ def ds_lrq_facet_values(
             filter_=filter, name=name, max_values=max_values
         ),
     )
-    return ds_lrq_run_loop(client=client, body=body)
+    tenant_details = get_tenant_related_payload(acc_conf)
+    return ds_lrq_run_loop(
+        logger, client=client, body=body, tenant_details=tenant_details
+    )
```

Comment:
Lets pass response from get_tenant_related_payload(acc_conf) to `PostQueriesLaunchQueryRequestBody.additional_properties` so we can avoid introducing additional attributes

Reply:
done